src/integration-test/groovy/au/org/ala/fieldcapture/ESPProjectSpec.groovy

```diff
@@ -32,7 +32,7 @@ class ESPProjectSpec extends StubbedCasSpec {
 
     def "generate Document PDF from document report tab"() {
         setup:
-        loginAsUser('1', browser)
```

Comment:
I'm a little concerned about this - did you need to change this because it stopped working?  If so maybe something is wrong with the logic?